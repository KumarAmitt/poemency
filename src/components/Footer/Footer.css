.footer {
    text-align: center;
    background-color: #3F51B5;
    margin: 0;
    padding: 0;
    color: #fff;
    padding: 0.5em;
}