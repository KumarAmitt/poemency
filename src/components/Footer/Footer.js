import React from 'react';

const Footer = () => (
  <div>
    <h2>&copy; Amit Kumar</h2>
  </div>
);

export default Footer;
