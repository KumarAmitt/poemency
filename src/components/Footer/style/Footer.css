.footer {
  text-align: center;
  background-color: #3f51b5;
  margin: 0;
  color: #fff;
  padding: 0.5em;
}
