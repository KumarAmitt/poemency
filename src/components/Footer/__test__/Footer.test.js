import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../Footer';

describe('Footer', () => {
  test('should match snapshot of Footer', () => {
    const tree = renderer.create(<Footer />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
