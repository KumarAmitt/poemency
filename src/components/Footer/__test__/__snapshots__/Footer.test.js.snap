// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match snapshot of Footer 1`] = `
<div
  className="footer"
>
  <h4>
    Â© Amit Kumar 2021
  </h4>
</div>
`;
